lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -172,52 +172,34 @@
 # Must clear as it will not override the existing puppet-lint rake task since we require to import for
 # the PuppetLint::RakeTask
 Rake::Task[:lint].clear
-# Relative is not able to be set within the context of PuppetLint::RakeTask
+# Utilize PuppetLint global configuration so that these settings can be tweaked by
+# spec_helper.rb in an individual module
 PuppetLint.configuration.relative = true
-PuppetLint::RakeTask.new(:lint) do |config|
-  config.fail_on_warnings = true
```

Comment:
This setting was lost in the refactor. I opened https://github.com/puppetlabs/puppetlabs_spec_helper/pull/347 to fix the regression.